scripts/start-setupForgeApiMods

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+: "${MODS_FORGEAPI_KEY:=}"
+: "${REMOVE_OLD_FORGEAPI_MODS:=false}"
+: "${MODS_FORGEAPI_PROJECTIDS:=}"
+: "${MODS_FORGEAPI_FILE:=}"
+: "${MODS_FORGEAPI_RELEASES:=RELEASE}"
+: "${REMOVE_OLD_MODS_DEPTH:=1} "
+: "${REMOVE_OLD_MODS_INCLUDE:=*.jar}"
+: "${VERSION:=LATEST}"
+
+#  FORGEAPI_BASE_URL used in manifest downloads below
+FORGEAPI_BASE_URL=${FORGEAPI_BASE_URL:-https://api.curseforge.com/v1}
+RELEASE_NUMBER_FILTER=1
+MINECRAFT_GAME_ID=432
+out_dir=/data/mods
+
+# shellcheck source=start-utils
+. "${SCRIPTS:-/}start-utils"
+isDebugging && set -x
+
+# Remove old mods/plugins
+if isTrue "${REMOVE_OLD_FORGEAPI_MODS}" && [ -z "${MODS_FORGEAPI_FILE}" ]; then
+  removeOldMods /data/mods
+fi
+
+# checks if no version or latest then grabs latest value from mojang.
+if [ ! "$VERSION" ] || [ $VERSION = "LATEST" ]; then
+  VERSION=$(curl -X GET -s "https://launchermeta.mojang.com/mc/game/version_manifest.json" | jq -jc ".latest.release")
+  if [ ! "$VERSION" ]; then
+    log "ERROR: Unable to retrieve version: ${VERSION} - from Mojang"
+    exit 2
+  fi
+fi
```

Comment:
Actually you can use `$VANILLA_VERSION` since it gets resolved and passed downstream